### ISSUE TITLE
Slurm broken

### ISSUE THREAD

**Issue by MilesCranmer** [JuliaParallel/ClusterManagers.jl#196]
Is anybody maintaining this package? I haven't been able to get Slurm working for the past month or so... It just ends up stalling on `connecting to worker 1 out of <N>`:

```julia
julia> p = addprocs_slurm(2)
connecting to worker 1 out of <N>
```

This is slurm 22.05.8. Not sure if this new version is breaking things or not.

**Comment by Moelf** [comment]
I don't have access to a slurm now, but it would be useful to know if a previous version was okay

**Comment by MilesCranmer** [comment]
Not sure I know how to test other versions of slurm... I am stuck with whatever my institute cluster has installed

**Comment by Moelf** [comment]
I guess I this case ask the HPC admin see if they know anything that might be the problem

**Comment by kescobo** [comment]
Ugh, @MilesCranmer that's annoying. I also don't currently have access to a SLURM cluster... this is the kind of thing that it would be nice if we had #105 that could test on different schedulers :facepalm: 

Would definitely be worth checking in with the cluster admin to see if SLURM was recently updated so we can at least know that that's the culprit.

**Comment by MilesCranmer** [comment]
We could if this PR gets finished https://github.com/JuliaParallel/ClusterManagers.jl/pull/193

**Comment by cnrrobertson** [comment]
@kescobo I can confirm that this issue started for me after an upgrade to Slurm on my institution's cluster. Unfortunately, I don't know what the previous version was, but currently the version is 23.11.1

**Comment by kescobo** [comment]
From the initial post, it looks like it goes back to v22

> This is slurm 22.05.8

Does anyone know if SLURM follows SemVer?

**Comment by DilumAluthge** [comment]
@MilesCranmer Does the [SlurmClusterManager.jl](https://github.com/kleinhenz/SlurmClusterManager.jl) package work for you?

**Comment by DilumAluthge** [comment]
Bump @MilesCranmer - I just wanted to check if the `SlurmClusterManager.jl` package works for you?

If so, I think we can add a note to the README recommending that users use `SlurmClusterManager.jl`, and then we can close this issue.

**Comment by MilesCranmer** [comment]
Yes, thanks (I think once the project activation stuff merges, it should be good to go)
